packages/cloud_firestore/cloud_firestore/macos/cloud_firestore.podspec

```diff
@@ -0,0 +1,32 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+require 'yaml'
+pubspec = YAML.load_file(File.join('..', 'pubspec.yaml'))
+libraryVersion = pubspec['version'].gsub('+', '-')
+
+Pod::Spec.new do |s|
+  s.name             = 'cloud_firestore'
+  s.version          = '0.0.1'
+  s.summary          = 'Firestore plugin for Flutter.'
+  s.description      = <<-DESC
+Firestore plugin for Flutter.
+                       DESC
+  s.homepage         = 'https://github.com/flutter/firestore'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Flutter Team' => 'flutter-dev@googlegroups.com' }
+  s.source           = { :path => '.' }
+  s.source_files = 'Classes/**/*'
+  s.public_header_files = 'Classes/**/*.h'
+  s.osx.deployment_target = '10.11'
```

Comment:
I think you also have to include `s.platform = :osx`

Reply:
It works well without. Would it help if we add it anyway?

Reply:
Maybe let's match what the [template](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/plugin/macos.tmpl/projectName.podspec.tmpl) is doing:

  s.platform = :osx, '10.11'

Reply:
Done